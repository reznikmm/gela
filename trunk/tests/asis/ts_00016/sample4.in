Associate -I../../../source/asis/spec -Isample sample/procedure_body-package_body_stub.adb
Action
Process_Context Compilation_Unit_Body    Procedure_Body.Package_Body_Stub
