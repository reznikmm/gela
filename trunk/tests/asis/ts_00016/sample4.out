Declaration_Kind:A_PACKAGE_BODY_DECLARATION Package_Body_Stub at procedure_body-package_body_stub.adb [2:1..9:22]
  Defining_Name_Kind:A_DEFINING_IDENTIFIER Package_Body_Stub at procedure_body-package_body_stub.adb [2:14..2:30]
  Declaration_Kind:A_PROCEDURE_BODY_DECLARATION X at procedure_body-package_body_stub.adb [4:4..7:7]
    Defining_Name_Kind:A_DEFINING_IDENTIFIER X at procedure_body-package_body_stub.adb [4:14..4:14]
    Statement_Kind:A_NULL_STATEMENT  at procedure_body-package_body_stub.adb [6:7..6:11]
